tools/src/main/java/com/nedap/archie/rules/evaluation/evaluators/functions/Round.java

```diff
@@ -33,7 +33,7 @@ public ValueList evaluate(List<ValueList> arguments) throws FunctionCallExceptio
         result.setType(PrimitiveType.Integer);
 
         for(Value valueObject : arguments.get(0).getValues()) {
-            result.addValue(new Value(Math.round((Double) valueObject.getValue())));
```

Comment:
This is not the correct solution to go from either a Long or a Double to a Double. Both are instances of Number, and that has a doubleValue() method that you should use.

See https://docs.oracle.com/javase/8/docs/api/java/lang/Number.html